### PR TITLE
raftstore: disable duplicated mvcc key compaction check by default

### PR DESCRIPTION
**Pull request by SpadeA-Tang**
### What is changed and how it works?

Issue Number: Close #15282

What's Changed:

```commit-message
disable duplicated mvcc key check compaction by default
```
It's better to let user to chose whether to use this.

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore/src/store/config.rs [hunk 1]
```diff
@@ -604,8 +608,11 @@ impl Config {
         if self.region_compact_check_step.is_none() {
             if raft_kv_v2 {
                 self.region_compact_check_step = Some(5);
+                self.region_compact_redundant_rows_percent = Some(20);
             } else {
                 self.region_compact_check_step = Some(100);
+                // Disable redundant rows check in defaul for v1.
```
Comment (tonyxuqqi) [inline components/raftstore/src/store/config.rs:614]: typo: defaul-> default

**Review by tonyxuqqi — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Pull request merged**